docs/keepers.md

```diff
@@ -0,0 +1,51 @@
+# Keepers
+
+## Overview
+
+## Common Conventions
+
+The Keeper class follows similar conventions as the smart contracts. See [smart_contracts.md](./smart_contracts.md) for explanations.
+
+## Classes
+### Keeper
+
+The base interface for all other keeper classes. `generate_actions_for_timestep` returns an array of `action` objects that is run by [experiment.py](../experiments/experiment.py)
+
+### VaultKeeper
+
+Class representation of a keeper who opens vaults. `open_max_vaults` will open the maximum possible number of vaults for each ilk.
```

Comment:
I would say "a keeper that owns vaults." it's not like their sole responsibility is just opening vaults, it's more so to differentiate from e.g. a bite keeper which doesn't need to do so

Reply:
Changed